src/test/java/org/cactoos/io/TeeInputStreamTest.java

```diff
@@ -67,4 +69,39 @@ public void copiesContentByteByByte() throws IOException {
         );
     }
 
+    @Test
+    public void leftInputUnclosed() {
+        try (StringWriterMock write = new StringWriterMock()) {
+            new LengthOf(
+                new TeeInput(
+                    "foo",
+                    new OutputTo(write)
+                )
+            ).intValue();
+            MatcherAssert.assertThat(
+                "Can''t use output after usage from TeeInput",
```

Comment:
@SharplEr Isn't the second `'` in `Can''t` redundant?

Reply:
@proshin-roman You are right.